src/mongocxx/options/find.hpp

```diff
@@ -308,6 +330,7 @@ class MONGOCXX_API find {
    private:
     stdx::optional<bool> _allow_partial_results;
     stdx::optional<std::int32_t> _batch_size;
+    stdx::optional<bsoncxx::document::view_or_value> _collation;
```

Comment:
Thats fair. Lets file it as such. I'm not sure why we didn't do it originally, TBH. Maybe we were just tired of pimpl'ing things at that point.